loadtesting/k6/tests/sendErc20.test.js

```diff
@@ -0,0 +1,49 @@
+import { ethClient, sendErc20ViaTransferFunction } from './utils/ethClient.js';
+import { createAccount, fundAccountFromFaucet } from './utils/accountManager.js';
+import { initialAccountBalance, erc20Address, erc20Owner } from './utils/consts.js';
+import { sendTokenOptions } from '../options/options.js';
+import { Trend, Counter } from 'k6/metrics';
+import { check } from 'k6';
+
+const sendErc20Requests = new Counter('send_erc20_requests');
+const sendErc20ErrorCounter = new Counter('send_erc20_errors');
+const sendErc20RequestTime = new Trend('send_erc20_request_time', true);
+
+export const options = sendTokenOptions;
+
+const pathToContractData = '../contracts/ERC20/ERC20';
+const abi = JSON.parse(open(pathToContractData + '.abi'));
+
+export default function sendErc20Test() {
+    // We need to have a unique signer account for each VU
```

Comment:
I think by this way we will get not real scenario
You can do transferring some tokens to all accounts as part of the preparation step 
and then sent tokens from different senders here